gen_cert.py

```diff
@@ -1841,8 +1834,32 @@ def fontlist_with_style(a_style):
         yOffset = minYOffset + ((max_height - height) / 2)
         paragraph.drawOn(PAGE, GUTTER_WIDTH - (name_style.fontSize / 12), yOffset)
 
+
+        # Parse Translations:
```

Comment:
This comment seems superfluous, given the number of times we reference "translation"/"translated" below.